app/src/main/java/io/appium/uiautomator2/core/AxNodeInfoHelper.java

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
autoformat?

Reply:
just added an empty line, because if exists in other modules and I find its presence better for readability :)